x-pack/test/functional/apps/infra/logs_source_configuration.ts

```diff
@@ -31,6 +31,16 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
     });
 
     describe('Allows indices configuration', () => {
+      const logPosition = {
```

Comment:
This extracted constant isn't strictly required, but I did it at first and liked how it tightned up the `navigateTo` calls below so leaving it in.

Reply:
And this is only used for the query params I guess, which are always in UTC?

Reply:
yep. In an early commit I thought I need to change those as well, but they're always UTC so no problem. 

I liked how `await logsUi.logStreamPage.navigateTo({ logPosition });` looked so I kept it :)

Thanks for the review! 🧡